### ISSUE TITLE
DropdownButtonFormField: Incorrect behavior dealing with labelText and hintText

### ISSUE THREAD

**Issue by Dynnosower** [flutter-form-builder-ecosystem/flutter_form_builder#1241]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package/Plugin version

^8.0.0

### Flutter doctor

<details>
  <summary>Flutter doctor</summary>

```bash
[√] Flutter (Channel stable, 3.7.12, on Microsoft Windows [Version 10.0.19044.2846], locale en-PH)
    • Flutter version 3.7.12 on channel stable at C:\Users\Dino\development\sdks\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4d9e56e694 (3 weeks ago), 2023-04-17 21:47:46 -0400
    • Engine revision 1a65d409c7
    • Dart version 2.19.6
    • DevTools version 2.20.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc1)
    • Android SDK at C:\Users\Dino\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.0-rc1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Users\Dino\AppData\Local\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop for Windows
    X Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[√] Android Studio (version 2022.2)
    • Android Studio at C:\Program Files\Android\Android Studio1
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-9586694)

[√] VS Code (version 1.78.0)
    • VS Code at C:\Users\Dino\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.62.0

[√] Connected device (4 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86    • Android 8.0.0 (API 26) (emulator)
    • Windows (desktop)                  • windows       • windows-x64    • Microsoft Windows [Version 10.0.19044.2846]
    • Chrome (web)                       • chrome        • web-javascript • Google Chrome 112.0.5615.139
    • Edge (web)                         • edge          • web-javascript • Microsoft Edge 112.0.1722.68

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

</details>

### Minimal code example

<details>
  <summary>Code sample</summary>

```dart
final List<String> _suffixes = ['Jr', 'Sr', "II", "III", "IV", "V"];

return FormBuilderDropdown(
                    name: "suffixes",
                    style: TextStyle(color: Colors.black),
                    decoration: InputDecoration(
                      contentPadding: EdgeInsets.all(13.5),
                      labelText: "Suffix",
                      hintText: "Select",
                    ),
                    items: _suffixes.map((suffix) {
                      return DropdownMenuItem(
                        value: suffix,
                        child: Text(suffix),
                      );
                    }).toList(),
                    onChanged: (value) {
                      print(value);
                    },
                  );

```

</details>

### Current Behavior

labelText and hintText are overlapping inside the borders of the field.

Screenshot:
![image](https://user-images.githubusercontent.com/82434323/237050909-8959381d-4a33-4811-a0f1-21e489cc88cf.png)

### Expected Behavior

labelText should float within the border while hintText is displayed within the field

Screenshot:
![image](https://user-images.githubusercontent.com/82434323/237051687-1986dc34-0aab-4d7a-a1b2-67953b893e83.png)

made using DropDownButtonFormField instead of FormBuilderDropdown and removing  name parameter

### Steps To Reproduce

1. Create flutter project
2. import flutter_form_builder dependency
3. flutter pub get
4. paste minimal code example provided within a screen in the flutter project

**Comment by deandreamatias** [comment]
Hi, thanks for open this issue
Do you can try used the preview version? (9.0.0-Dev.1) and verify the behavior again?  Thanks

**Comment by Dynnosower** [comment]
The issue is resolved in version 9.0.0-Dev.1. Thank you.

**Comment by Dynnosower** [comment]
The issue is resolved in version 9.0.0-Dev.1. Thank you

**Comment by deandreamatias** [comment]
I will release this version for all on this month. Until that, can use the prerelease
